packages/@aws-cdk/aws-cloud9/lib/environment.ts

```diff
@@ -61,6 +62,29 @@ export interface Ec2EnvironmentProps {
    * @default - no description
    */
   readonly description?: string;
+
+  /**
+   * The AWS CodeCommit repository to be cloned
+   *
+   * @default - do not clone any repository
+   */
+  readonly repositories?: IRepository[];
+}
+
+/**
+ * The interface of the codecommit repository for Cloud9 environment
+ */
+export interface IRepository {
```

Comment:
Since this interface is what JSII calls a 'struct' (it only contains properties, not methods), it should not have the `I` prefix (also, this name is super confusing with CodeCommit's `IRepository`). How about `Cloud9Repository` instead?